state/machine_linklayerdevices.go

```diff
@@ -293,75 +293,13 @@ func (m *Machine) prepareOneSetLinkLayerDeviceArgs(
 ) (_ *linkLayerDeviceDoc, err error) {
```

Comment:
With this removal the method body looks trivial. Assuming it is not called from different call-sites, should we inline it in the future?

Reply:
I'll be deleting it once the tests use an alternate method for priming link-layer data.